server/src/main/java/org/elasticsearch/TransportVersions.java

```diff
@@ -254,6 +254,7 @@ static TransportVersion def(int id) {
     public static final TransportVersion ML_INFERENCE_ELASTIC_DENSE_TEXT_EMBEDDINGS_ADDED_8_19 = def(8_841_0_59);
     public static final TransportVersion ML_INFERENCE_COHERE_API_VERSION_8_19 = def(8_841_0_60);
     public static final TransportVersion ESQL_DOCUMENTS_FOUND_AND_VALUES_LOADED_8_19 = def(8_841_0_61);
+    public static final TransportVersion ESQL_PROFILE_INCLUDE_PLAN_8_19 = def(8_841_0_62);
```

Comment:
This needs to be added in a PR to main first that include the `isPatchFrom` logic to parse it.

Reply:
Yeap, opened a pr for that: https://github.com/elastic/elasticsearch/pull/130187